### ISSUE TITLE
Talkback does not announce dates correctly for English United Kingdom dd/mm/yy format

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#343]
```
What steps will reproduce the problem?
1. Install an App such as the BBC iPlayer (UK)
2. Get Talkback to announce programme details
3. Watch for dates in the App in format dd/mm/yy

What is the expected output? What do you see instead?
In the UK dates use the format dd/mm/yy. Talkback assumes dates are mm/dd/yy so 
announces the wrong date. This happens even when the language is set to English 
United Kingdom.


What version of the product are you using? On what operating system?
Android 4.3. Talkback 3.5.1.

Please provide any additional information below.




```

Original issue reported on code.google.com by `mossieg...@gmail.com` on 14 Jan 2014 at 3:25

**Comment by GoogleCodeExporter** [comment]
```
I do have date format set to be day/month/year in my date/time settings.

In the BBC iPlayer App choose the "Most Popular" link and scroll down the 
programms until yu find entries with dates shown as dd/mm/yyyy.

NB some dates are announced correctly, other are not. It depends if the date 
makes sense in US mm/dd/yyyy format eg. 12/1/2014 is announced as December 1 
2014, 13/1/2014 is announces as 13 January 2014.
```

Original comment by `mossieg...@gmail.com` on 15 Jan 2014 at 3:14
